hstest/testing/runner/sql_runner.py

```diff
@@ -46,9 +46,11 @@ def parse_sql_file(self) -> None:
         with open(file_path, 'r') as file:
             lines = file.readlines()
             sql_content = " ".join(lines).replace("\n", "")
-            commands = re.findall("(\\w+)\\s+?=\\s+?\"(.+?)\"", sql_content)
+            commands = re.findall(r"(\w+)\s+?=\s+?\"(|.+?)\"", sql_content)
```

Comment:
Кажется вот этот код будет эквивалентным:
```suggestion
            commands = re.findall(f'(\w+)\s+?=\s+?"(.*?)"', sql_content)
```

Reply:
А ты запускал тест с этой регуляркой? Он успешно завершился?

Reply:
Проверил, да все ок, `*` решает проблему. Сейчас обновлю.